.github/workflows/atlantis-image.yml

```diff
@@ -146,6 +151,12 @@ jobs:
         target: ${{ matrix.image_type }}
         labels: ${{ steps.meta.outputs.labels }}
         outputs: type=image,name=target,annotation-index.org.opencontainers.image.description=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.description'] }}
+    - name: "Sign and Attest Image"
+      uses: actions/attest-build-provenance@7668571508540a607bdfd90a87a560489fe372eb # v2.1.0
+      with:
+        subject-path: ${{ steps.build.outputs.name }}
```

Comment:
```suggestion
        subject-path: ${{ steps.build.outputs.image-name }}@${{ steps.build.outputs.digest }}
```

Reply:
We need to add a ```ìd: build``` to the build step in order to be able to reference it like that.